mm2src/coins_activation/src/eth_with_token_activation.rs

```diff
@@ -477,5 +487,17 @@ fn eth_priv_key_build_policy(
             Ok(EthPrivKeyBuildPolicy::Metamask(metamask_ctx))
         },
         EthPrivKeyActivationPolicy::Trezor => Ok(EthPrivKeyBuildPolicy::Trezor),
+        EthPrivKeyActivationPolicy::WalletConnect => {
+            let wc = WalletConnectCtx::from_ctx(ctx).expect("WalletConnectCtx should be initialized by now!");
```

Comment:
Related to the previous comment, we shouldn't use expect here in case we removed initialization on mm2 start.

Reply:
added todos